### ISSUE TITLE
Get (some) ACAM headers from FPoffsets

### ISSUE THREAD

**Issue by chazshapiro** [CaltechOpticalObservatories/NGPS#210]
This is dict now available from FPoffsets:

```
acam_params={
    "PIXSCALE"  : pixscale,        # arcsec/px
    "CDELT1"    : pixscale/3600.,  # deg/px
    "CDELT2"    : pixscale/3600.,   # deg/px
    }
```

These are part of the geometric model and belong with their brethren geometric parameters.

**Comment by chazshapiro** [comment]
also get CRPIX1 and CRPIX2

**Comment by astronomerdave** [comment]
implemented in commit 69c9618de836712fcdcdfe5fd0de6a75dd9cf431
